src/QuickMarc.js

```diff
@@ -4,15 +4,21 @@ import {
   Switch,
   Route,
 } from 'react-router-dom';
+
 import { IfPermission } from '@folio/stripes/core';
+import { CommandList } from '@folio/stripes-components';
```

Comment:
It needs to be changed to `@folio/stripes/components` to put away the `Eslint` error.

Reply:
Thanks for noticing, fixed